src/theme.rs

```diff
@@ -475,9 +475,9 @@ pub(crate) enum FooterStyle {
 impl Default for FooterStyle {
     fn default() -> Self {
         Self::Template {
-            left: Some("{current_slide} / {total_slides}".to_string()),
-            center: None,
-            right: None,
+            left: Some("{author}".to_string()),
+            center: Some("{title}".to_string()),
```

Comment:
This is too verbose of a default, I wouldn't change this at all. e.g. I don't think people should see my name in every slide + the title of the presentation can be too long and cause overflows.

If you want to use this as your default, you can set it globally for all your presentations https://mfontanini.github.io/presenterm/guides/configuration.html#default-theme. you can define a custom theme with these properties and use that as the default. This tool allows you to configure as much as possible so you can tweak it as much as you need without the tool enforcing anything on you

Reply:
I removed